src/components/ShippingInfo/ShippingInfo.jsx

```diff
@@ -55,21 +54,22 @@ function ShippingInfo() {
           orderId,
         },
       });
+
+      // TO DO - Change to styled alert
       alert('Shipping Successful!');
+
       history.push('/samples');
     }
-  } // end handleSubmit
-
-  const backBtn = () => {
```

Comment:
Why are we nixing this function?

Reply:
The only thing it needs to do is a history.push, so I just moved that inline on the onClick

Reply:
(line 127)

Reply:
(line 127)